fastly/resource_fastly_service_dictionary_items_v1.go

```diff
@@ -0,0 +1,219 @@
+package fastly
+
+import (
+	"fmt"
+	gofastly "github.com/fastly/go-fastly/fastly"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceServiceDictionaryItemsV1() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceServiceDictionaryItemsV1Create,
+		Read:   resourceServiceDictionaryItemsV1Read,
+		Update: resourceServiceDictionaryItemsV1Update,
+		Delete: resourceServiceDictionaryItemsV1Delete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"service_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "Service Id",
+			},
+
+			"dictionary_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "Dictionary Id",
+			},
+
+			"items": {
+				Type:        schema.TypeMap,
+				Description: "Dictionary Items",
+				Optional:    true,
+			},
+		},
+	}
+}
+
+func resourceServiceDictionaryItemsV1Create(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*FastlyClient).conn
+
+	serviceID := d.Get("service_id").(string)
+	dictionaryID := d.Get("dictionary_id").(string)
+	items := d.Get("items").(map[string]interface{})
+
+	for k, v := range items {
+		_, err := conn.CreateDictionaryItem(&gofastly.CreateDictionaryItemInput{
+			Service:    serviceID,
+			Dictionary: dictionaryID,
+			ItemKey:    k,
+			ItemValue:  v.(string),
+		})
+
+		if errRes, ok := err.(*gofastly.HTTPError); ok {
+			if errRes.StatusCode != 409 {
+				return err
+			}
+		} else if err != nil {
+			return err
+		}
+	}
+
+	// TODO review this
+	d.SetId(fmt.Sprintf("%s/%s", serviceID, dictionaryID))
```

Comment:
As per the above `TODO`; what needs to be reviewed? Whether this is an acceptable ID?

Reply:
This was concerning the structure of the ID.  The serviceID and dictionaryID are not being derived from this value and so the resource ID is not being sent to the remote state.  I have removed the TODO